src/ontology/edits/oeo-shared.omn

```diff
@@ -73,20 +70,23 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/523",
 AnnotationProperty: OEO_00020425
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "An oekg specific annotation that indicates how an entity should be used by the OEKG.",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1529
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1529
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1897",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An oekg specific annotation that indicates how an entity should be used by the OEKG.",
         rdfs:comment "allowed value: \"study descriptor\"",
         rdfs:label "oekg annotation"
     
     
 AnnotationProperty: OEO_00020426
 
     Annotations: 
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1592
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652
```

Comment:
I wonder about those two rows:
```
OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1592
pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652
```

The AnnotationProperty itself was introduced only in issue 1850 / PR 1922, so why reference here issues 1592 and 1652?

It was part of the editor note, but there it does not make sense either. 
So delete these two lines completely?

Reply:
I guess, I added the wrong issue and pr number in the first place. The annotation was added in pr #1913